### ISSUE TITLE
ABGT 5.6 Beta Null Pointer Exceptions

### ISSUE THREAD

**Issue by hiroki-o** [Unity-Technologies/AssetGraph#48]
**[Original report](https://bitbucket.org/Unity-Technologies/assetbundlegraphtool/issue/13) by Sam Narain (Bitbucket: [samnarain](https://bitbucket.org/samnarain), GitHub: [samnarain](https://github.com/samnarain)).**

----------------------------------------

A variety of Null Pointer exceptions occur with the ABGT tool in 5.6.0 B3. The GraphEditor window will not populate after spawning its window.

Tailing Editor.log indicates EditorGUIUtilityBindings.gen.cs:31 has the culprit. Here is an excerpt of the output.

```
#!txt

(Filename: /Users/builduser/buildslave/unity/build/artifacts/generated/common/editor/EditorGUIUtilityBindings.gen.cs Line: 318)

NullReferenceException: Object reference not set to an instance of an object
  at UnityEditor.EditorGUIUtility.TempContent (System.String[] texts) [0x00001] in /Users/builduser/buildslave/unity/build/artifacts/generated/common/editor/EditorGUIUtilityBindings.gen.cs:318 
  at UnityEditor.EditorGUI.Popup (Rect position, Int32 selectedIndex, System.String[] displayedOptions, UnityEngine.GUIStyle style) [0x00015] in /Users/builduser/buildslave/unity/build/artifacts/generated/common/editor/EditorGUIBindings.gen.cs:552 
  at UnityEditor.EditorGUILayout.Popup (Int32 selectedIndex, System.String[] displayedOptions, UnityEngine.GUIStyle style, UnityEngine.GUILayoutOption[] options) [0x00019] in /Users/builduser/buildslave/unity/build/Editor/Mono/EditorGUI.cs:6356 
  at AssetBundleGraph.AssetBundleGraphEditorWindow.DrawGUIToolBar () [0x00163] in /Users/sam/Projects/UNITY/Testing Ground/Testing Ground/Assets/AssetBundleGraph/Editor/GUI/AssetBundleGraphEditorWindow.cs:571 
  at AssetBundleGraph.AssetBundleGraphEditorWindow.OnGUI () [0x00002] in /Users/sam/Projects/UNITY/Testing Ground/Testing Ground/Assets/AssetBundleGraph/Editor/GUI/AssetBundleGraphEditorWindow.cs:820 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x000d0] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222 
```

* OSX 10.11.6, using the editor with force-metal.

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

This looks to me that issue is fixed in commit 42b5fa7. Can you pull the latest and see if the issue still happens?

**Comment by hiroki-o** [comment]
**Original comment by Sam Narain (Bitbucket: [samnarain](https://bitbucket.org/samnarain), GitHub: [samnarain](https://github.com/samnarain)).**

----------------------------------------

The Null Pointer Exceptions are resolved with versions that include commit 42b5fa7
